analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -195,7 +196,20 @@ abstract class PackageManager {
                 } catch (e: Exception) {
                     e.showStackTrace()
 
-                    result[definitionFile] = ProjectAnalyzerResult(Main.allowDynamicVersions, Project.EMPTY,
+                    val definitionFilePath = VersionControlSystem.getPathToRoot(definitionFile) ?: ""
+
+                    val errorProject = Project.EMPTY.copy(
+                            id = Identifier(
+                                    provider = Unmanaged.toString(),
+                                    namespace = "",
+                                    name = definitionFilePath.takeIf { it.isNotBlank() } ?: definitionFile.name,
```

Comment:
Can't we simply use `definitionFile`? `definitionFile` should be relative here, so it's either the full relative path to the scan root (which should equal the VCS root) or, if the file is in the scan root, the name of the file.

Reply:
Done.